recipes/sawfish/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Sawfish already auto-extracts to bin directory, so no need to move it there.
```

Comment:
There is no need of the build.sh file. You could remove it!

Reply:
Thanks, I tried removing but no luck, however I was able to change it to a simple cp and it works now.